react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -496,6 +508,19 @@ public void onFocusChange(View v, boolean hasFocus) {
         // in case check ReactTextInputManager
     }
 
+    @Override
+    public void updateExtraData(ReactAztecText view, Object extraData) {
+        if (extraData instanceof ReactTextUpdate) {
+            ReactTextUpdate update = (ReactTextUpdate) extraData;
+
+            view.setPadding(
```

Comment:
I'm getting this warning in AS about these lines for RTL compatibility:

```
For RtL compatibility, use setPaddingRelativeor ViewCompat.setPaddingRelative() when setting left/right padding. less... (Ctrl+F1) 
Inspection info:For RtL compatibility, use setPaddingRelativeor ViewCompat.setPaddingRelative() when setting left/right padding.
```

Reply:
This might be a local linting issue on my end.. I couldn't determine why it gave this warning when `ReactTextUpdate` merely inherits from `java.lang.Object` :thinking:

Reply:
Good point. I've been actually mirroring the code in upstream's implementation of the TextInput shadow node here: https://github.com/facebook/react-native/blob/d88e4701fc46b028861ddcfa3e6ffb141b3ede3d/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java#L193-L197

I'm kinda torn but, I think it makes more sense at the moment to follow upstream's lead on this one. Alternatively, we can even completely remove the code to set the padding since it's not a real goal for this PR anyway. WDYT?

Reply:
> I think it makes more sense at the moment to follow upstream's lead on this one.

I agree, this seems like the best way forward. I haven't tested many scenarios without the code (only the "Lorem ipsum" test in the description), and I'm unsure if there are scenarios where it would be important to have it. So keeping it close to upstream seems the sensible approach. LGTM!